<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fenetre Principale</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-office">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(152, 152, 152);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="Results_Interaction">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>401</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Collect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Collecter Ensemble Interactions</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>160</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="1">
      <widget class="QDateEdit" name="Date1"/>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="Date2"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Intervalle De Date: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>196</width>
      <height>146</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="1" column="1">
      <widget class="QDateEdit" name="Date1_Todo"/>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="Date2_Todo"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Intervalle De Date: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="IdContact_Box">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="checkContact">
       <property name="text">
        <string>Selectionnez Ce Contact</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Collect_Todo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Collecter Ensemble Todos</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_Contacts">
    <property name="title">
     <string>A Propos Des Contact</string>
    </property>
    <addaction name="actionAjouter_Un_Contact_2"/>
    <addaction name="actionLister_Les_Contacts_2"/>
   </widget>
   <widget class="QMenu" name="menuRequete">
    <property name="title">
     <string>Requete</string>
    </property>
    <addaction name="actionExport_Json"/>
   </widget>
   <addaction name="menu_Contacts"/>
   <addaction name="menuRequete"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAjouter_Un_Contact_2"/>
   <addaction name="actionLister_Les_Contacts_2"/>
   <addaction name="actionExport_Json"/>
  </widget>
  <action name="actionAjouter_Un_Contact_2">
   <property name="icon">
    <iconset resource="Ressources_QT.qrc">
     <normaloff>:/rec/Ressources/add-user.png</normaloff>:/rec/Ressources/add-user.png</iconset>
   </property>
   <property name="text">
    <string>tr(&quot;Ajouter Un Contact&quot;)</string>
   </property>
  </action>
  <action name="actionLister_Les_Contacts_2">
   <property name="icon">
    <iconset resource="Ressources_QT.qrc">
     <normaloff>:/rec/Ressources/listing-user.png</normaloff>:/rec/Ressources/listing-user.png</iconset>
   </property>
   <property name="text">
    <string>tr(&quot;Lister Les Contacts&quot;)</string>
   </property>
  </action>
  <action name="actionExport_Json">
   <property name="icon">
    <iconset resource="Ressources_QT.qrc">
     <normaloff>:/rec/Ressources/json.png</normaloff>:/rec/Ressources/json.png</iconset>
   </property>
   <property name="text">
    <string>Export Json</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Ressources_QT.qrc"/>
 </resources>
 <connections/>
</ui>
